lib/index.js.flow

```diff
@@ -0,0 +1,77 @@
+/* @flow */
+type ConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function>(
+  mapStateToProps: (state: Object) => SP,
```

Comment:
mapStateToProps can have additional parameter - own props of the connected component.